packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -1299,6 +1299,101 @@ describe('@api RPC Server Acceptance Tests', function () {
                 const res = await relay.call('eth_call', [callData, 'latest'], requestId);
                 expect(res).to.eq(BASIC_CONTRACT_PING_RESULT);
             });
+
+            it('should fail to execute "eth_call" with correct block number', async function () {
```

Comment:
It looks like this test is supposed to succeed rather than fail?

Reply:
Fixed.